spec/javascripts/NewService/formElements/__snapshots__/ServiceManualListItems.spec.tsx.snap

```diff
@@ -106,11 +109,18 @@ exports[`should render required form fields with errors 1`] = `
       type="text"
       value="new api"
     />
-    <p
-      className="inline-errors"
-    >
-      Only ASCII letters, numbers, dashes and underscores are allowed.
-    </p>
+    <HelperTextItem>
+      <p
+        className="inline-errors"
+      >
+        Only ASCII letters, numbers, dashes and underscores are allowed.
+      </p>
+      <p
+        className="inline-hints"
+      >
+        Only ASCII letters, numbers, dashes and underscores are allowed.
+      </p>
+    </HelperTextItem>
```

Comment:
This shouldn't be duplicated.

Reply:
I tried it removing intentionally and then I ran command to update jest so it add it again 
so I don't know how to remove it from snapshot.

Reply:
The error message is defined for the test. Since we changed the message in the server, now this is outdated. Change the following prop: https://github.com/3scale/porta/blob/bc7513d8844cc3dd3519c16ccdd1b60750f22b60/spec/javascripts/NewService/formElements/ServiceManualListItems.spec.tsx#L28

Reply:
Okay

Reply:
resolved